src/wazuh_db/helpers/wdb_agents_helpers.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Wazuh SQLite integration
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ * February 17, 2021.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "wdb_agents_helpers.h"
+//#include "defs.h"
+#include "wazuhdb_op.h"
+
+#define WDBQUERY_SIZE OS_BUFFER_SIZE
```

Comment:
I decided to move the definition to the `wazuhdb_op.h` header because is the one that contains the definitions of the helpers to be communicated with Wazuh DB.

Fixed